src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormAclControllerImpl.java

```diff
@@ -9,18 +9,20 @@
 import com.techsophy.tsf.runtime.form.service.FormAclService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.bind.annotation.RestController;
-
 import static com.techsophy.tsf.runtime.form.constants.FormAclConstants.NO_RECORD_FOUND;
 
 @RestController
 @RequiredArgsConstructor(onConstructor_ = {@Autowired})
+@Transactional
 public class FormAclControllerImpl implements FormAclController {
     
     private final FormAclService formAclService;
     private final GlobalMessageSource globalMessageSource;
+    
     @Override
-    public ApiResponse<FormAclDto> saveFormAcl(FormAclDto formAclDto) throws JsonProcessingException {
+    public ApiResponse<FormAclDto> saveFormAcl(FormAclDto formAclDto) {
```

Comment:
Please add the transactional annotation

Reply:
in both post/delete methods

Reply:
yes vaibhav I have added it at the class level so there is no need to add on each method